views/panel.php

```diff
@@ -59,8 +65,12 @@
     window.fiber = json;
   </script>
 
-  <?php foreach ($assets['js'] as $js): ?>
-  <?= Html::tag('script', '', $js) . PHP_EOL ?>
+  <?php foreach ($assets['js'] as $key => $js): ?>
+    <?php if ($key === 'index'): ?>
+    <?= Html::tag('script', [$assets['imports'], "import('{$js['src']}')"], A::without($js, 'src')) . PHP_EOL ?>
```

Comment:
This is super hard to read and understand. I think it would be better to not use `Html::tag` here but really write out the script tag.